peachjam/resources.py

```diff
@@ -150,12 +152,21 @@ def after_import_row(self, row, instance, row_number=None, **kwargs):
                 judge, _ = Judge.objects.get_or_create(name=judge)
                 judgment.judges.add(judge)
 
-        for case_number in row["case_numbers"]:
-            if case_number["number"]:
-                MatterType.objects.get_or_create(name=case_number["matter_type"])
-                CaseNumber.objects.create(
-                    document=judgment,
-                    number=case_number["number"],
-                    year=case_number["year"],
-                    matter_type=MatterType.objects.get(name=case_number["matter_type"]),
-                )
+        if "string_override" in row:
+            CaseNumber.objects.create(
+                string_override=row["string_override"], document=judgment
+            )
+        if "case_numbers" in row:

```

Comment:
These must be on the same object. Otherwise we'll get one for the string override, and one for the split out details, which is two case numbers.